service/controller/v1/prometheus/scrapeconfig.go

```diff
@@ -944,6 +944,12 @@ func getScrapeConfigs(service v1.Service, metaConfig Config) []config.ScrapeConf
 				Scheme:                 HttpsScheme,
 				HTTPClientConfig:       secureHTTPClientConfig,
 				ServiceDiscoveryConfig: etcdStaticConfig,
+
+				RelabelConfigs: []*relabel.Config{
+					MetricRelabelConfigs: []*relabel.Config{
```

Comment:
This should be its own member of the `ScrapeConfig`, not nested in `RelabelConfigs` -- I don't think you need `RelabelConfigs` at all in this case

Reply:
oops thanks for pointing that out!